underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java

```diff
@@ -153,10 +153,20 @@ public S3UnderFileSystem(AlluxioURI uri) throws ServiceException {
     mClient = new RestS3Service(awsCredentials, null, null, props);
     mBucketPrefix = PathUtils.normalizePath(Constants.HEADER_S3N + mBucketName, PATH_SEPARATOR);
 
+    mAccountOwnerId = mClient.getAccountOwner().getId();
+    // Gets the owner from user-defined static mapping from S3 canonical user id  to Alluxio
+    // user name.
+    String owner = CommonUtils.getValueFromStaticMapping(
+        Configuration.get(PropertyKey.UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING),
+        mAccountOwnerId);
+    // If there is no user-defined mapping, use the display name.
+    if (owner == null) {
+      owner = mClient.getAccountOwner().getDisplayName();
+    }
+    mAccountOwner = owner == null ? "" : owner;
```

Comment:
Is defaulting to `""` safe, since all the defaults will go to `""`? Would it be better to use the `id`?